lib/caps.js

```diff
@@ -17,6 +17,7 @@ const APPS = {
   'iOSHybridWK': 'http://appium.s3.amazonaws.com/FLWebView.app.zip',
   'Android': 'http://appium.s3.amazonaws.com/ContactManager.apk',
   'AndroidHybrid': 'http://appium.s3.amazonaws.com/ApiDemos-debug-2015-03-19.apk',
+  'AndroidLoad': 'http://www.saucelabs.com/example_files/stressapptest.apk',
```

Comment:
can we put this up on s3 somewhere? probably more reliable than the sauce website?

Reply:
I'll get that moved and updated.